### DISCUSSION TITLE
Loading a list of URLs as JSON file

### DISCUSSION THREAD

**Discussion by asitemade4u** [MontFerret/ferret#665] [Q&A] [answered]
I am stuck with the issue #488, even when not using CDP. But replacing strings in any editor is a cinch and I can indeed save the list of pages into a JSON document, after revising it on a text editor.
How would you recommend coding a script so that Ferret accepts for input a list of URLs as a JSON document, then scrapes every item in the list?

**Comment by ziflex** [comment]
You can pass the list using parameters.
https://www.montferret.dev/docs/fql/bind-parameters/

**Comment by 3timeslazy** [comment, marked as answer]
@asitemade4u try this:
* urls.json
```json
[ "https://url.one", "https://url.two", ... ]
```
* script.fql
```
USE IO::FS

LET input = READ("urls.json")

LET urls = JSON_PARSE(input)

FOR url IN urls
    DO_SMTH(url)
```
